fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/version/VersionHistoryService.java

```diff
@@ -98,6 +98,7 @@ public void init() {
     private void getLatestVersionHistoryForSchema() {
         // Note how we don't care about connections here...that is all
         // hidden inside the target adapter implementation
+        versionHistoryMap.clear();
```

Comment:
Why are you. clearing this?

Reply:
Because the database table contains the "truth" and we are about to read all of it. The data in the database should replace everything in the map, which is just being used as a cache.